docs/installation/compatibility.md

```diff
@@ -1,64 +1,46 @@
-# Ensuring compatibility with legacy frameworks
+# Use GOV.UK Frontend with old frameworks or colours
 
-GOV.UK Frontend has been designed to work alongside the 'legacy frameworks' -
-[GOV.UK Frontend Toolkit], [GOV.UK Template] and [GOV.UK Elements].
+You can configure GOV.UK Frontend to work with the following 'legacy' frameworks:
 
-If you are using components from GOV.UK Frontend within a page that also
-includes any of the legacy frameworks, you should [enable compatibility
-mode](#compatibility-mode).
+- [GOV.UK Frontend toolkit]
+- [GOV.UK Template]
+- [GOV.UK Elements]
 
-If a part of your service uses GOV.UK Frontend without including the legacy
-frameworks, but you need to provide a consistent experience with these
-frameworks for your users, you can configure GOV.UK Frontend to [use the legacy
-colour palette](#using-the-legacy-colour-palette).
+You cannot use the following features if you installed GOV.UK Frontend from `dist`. [Install with node package manager](installing-from-npm.md) instead.
 
-Enabling either of these options means that you are no longer benefiting from
-many of the accessibility and usability improvements made in GOV.UK Frontend, as
-it prioritises providing a consistent experience for the user.
+If you use these features, your service will not have GOV.UK Frontend's many accessibility and usability improvements.
 
-In order to enable either of these options you'll need to have [installed GOV.UK
-Frontend using npm](installing-with-npm.md). These features are not available if
-you are using the 'dist' version.
+## Turn on 'compatibility mode'
 
+You can turn on 'compatibility mode' if you want to use both GOV.UK Frontend components and a legacy framework in a service.
 
-## Compatibility mode
+GOV.UK Frontend components in your pages will:
```

Comment:
It's not just the components – this would also apply to anything styled using GOV.UK Frontend, for example headings, paragraphs and links.

Reply:
Ah good to know, thanks - tweaked in https://github.com/alphagov/govuk-frontend/pull/1439/commits/f6214966822764e76468e3e5d4b672f9c4796c7f.